expo/gbe/templates/gbe/landing_page.tmpl

```diff
@@ -18,6 +18,7 @@
                     {% if profile and tickets %}
                     <div class='landing_box'>
 			 {% include 'gbe/incl_tickets.tmpl' %}
+       {% include 'gbe/profile/incl_performer.tmpl' %}
```

Comment:
It looks like the template indentation is messed up here - not your code, so not a blocker but this is a good opportunity to do some cleanup if you want. (1) consistent 4-space indents throughout (2) eliminate tabs, use spaces only.